source/runbooks/creating-new-private-dns-zones.html.md.erb

```diff
@@ -0,0 +1,61 @@
+---
+owner_slack: "#modernisation-platform"
+title: Creating new private DNS zones
+last_reviewed_on: 2023-07-26
+review_in: 6 months
+---
+
+# <%= current_page.data.title %>
+
+Privaye DNS zones are created in the Modernisation Platform `core-network-services` account through Terraform.
+We provide `modernisation-platform.service.justice.gov.uk` and `modernisation-platform.internal` to customers on a per-business unit basis.
+
+This new service allows for the creation of custom private DNS zones this has to be done in to be done in two pull requests the first one to create the zone and a second extend the zone to the VPCs
```

Comment:
`private DNS zones. This` instead of `private DNS zones this`?

Reply:
corrected